cloudhub/modules/ROOT/pages/vpc-tutorial.adoc

```diff
@@ -51,6 +51,10 @@ Don't associate Anypoint VPC with a design environment. You can deploy apps to t
 There is a cache timeframe between services that prevents rate limiting errors with different time configuration across Anypoint control planes.
 If you create an environment in Access Management and attempt to immediately use it to create or edit a VPC in Runtime Manager, you might experience a delay to see the newly created environment in the VPCs *Environments* dropdown.
 
+[NOTE]
+If you create a new VPC or update an existing one and associate an environment that is used in another VPC in the same region, any applications deployed in that other VPC are automatically deployed in the new or updated VPC as soon as it is restarted or updated.
+
```

Comment:
I found this very hard to parse. Writer wasn't very helpful. Here's what I think you're saying. Is this right?

If you associate an application with a VPC that shares an environment with another VPC in the same region, applications deployed in one are automatically deployed to the other when the other is restarted or updated.